src/domain/events/eventsPage.module.scss

```diff
@@ -46,7 +46,7 @@
     grid-gap: var(--grid-gap);
 
     @include respond_above(sm) {
-      grid-template-columns: 1fr var(--width-input);
+      grid-template-columns: var(--width-input) 1fr auto;
```

Comment:
the free text search field is quite narrow, but maybe it's still better like this

Reply:
I don't think it was any wider previously